x-pack/legacy/plugins/siem/public/components/alerts_viewer/translations.ts

```diff
@@ -17,3 +17,20 @@ export const TOTAL_COUNT_OF_ALERTS = i18n.translate('xpack.siem.hosts.totalCount
 export const ALERTS_TABLE_TITLE = i18n.translate('xpack.siem.hosts.alertsDocumentType', {
   defaultMessage: 'Alerts',
 });
+
+export const ALERTS_STACK_BY_MODULE = i18n.translate(
+  'xpack.siem.histogram.alertsStackByOptions.eventActions',
+  {
+    defaultMessage: 'modules',
```

Comment:
`ALERTS_STACK_BY_MODULE` is used here:

```
const alertsStackByOptions: MatrixHistogramOption[] = [
  {
    text: i18n.ALERTS_STACK_BY_MODULE,
    value: 'event.module',
  },
];
```

so it looks like `defaultMessage: 'modules'` should also be non-plural: `defaultMessage: 'module',`